lib/id3c/cli/command/etl/redcap_det.py

```diff
@@ -100,11 +88,21 @@ def decorator(routine: Callable[..., Optional[dict]]) -> click.Command:
             help        = "Write the output FHIR documents to stdout. You will likely want to redirect this to a file",
             default     = False)
 
+        # XXX FIXME: I don't think @command_for_project should even be
+        # providing the "cache" parameter.  What is cached and where it is stored is
+        # something specific to each REDCap DET routine, not a global invariant.
+        #   -trs, 19 Dec 2019
+        @click.option("--cache-file",
+            metavar = "<cache.pickle>",
+            help = "Local cache file for storing address lookups. Must be specified to cache lookups",
+            required = False,
+            type = click.Path())
```

Comment:
Good use of `click.Path()`. It'd be nice to override [its defaults](https://click.palletsprojects.com/en/7.x/api/#click.Path) that aren't applicable here:

```
click.Path(dir_okay = False, writable = True)
```